src/lib/alloc.c

```diff
@@ -592,8 +600,18 @@ static void alloc_trace_heap(enum mem_zone zone, uint32_t caps, size_t bytes)
 		heap_base = memmap->buffer;
 		heap_count = PLATFORM_HEAP_BUFFER;
 		break;
+#if CONFIG_CORE_COUNT > 1
+	case SOF_MEM_ZONE_RUNTIME_SHARED:
+		heap_base = memmap->runtime_shared;
+		heap_count = PLATFORM_HEAP_RUNTIME_SHARED;
+		break;
+	case SOF_MEM_ZONE_SYS_SHARED:
+		heap_base = memmap->system_shared;
+		heap_count = PLATFORM_HEAP_SYSTEM_SHARED;
+		break;
+#endif
 	default:
-		tr_err(&mem_tr, "alloc trace: unsupported mem zone");
+		tr_warn(&mem_tr, "alloc trace: unsupported mem zone");
```

Comment:
Try to use unknown memory zone seems to be more like error rather than warning.

Reply:
um, for this unsupported zone, we actually change to use another zone for it so not fatal, let me change and make this clear.

Reply:
> for some unsupported zones like runtime_shared for single core, we actually change to use another zone for it so not fatal

In this scenario this message won't even be displayed. This is the scenario eg. when somebody use different enum-set to choose memory zone, so this may be logical error in code.

Reply:
@ktrzcinx check my latest update, you will get what I meant.

Reply:
But anyway, I added the runtime_shared and system_shared zone type coverage and keep logging errors in the default case, in the latest update.